client/src/options.js

```diff
@@ -11,3 +11,35 @@ export const dayOptions = [
   { value: 'tomorrow', label: 'Tomorrow' },
 ];
 export const dayDefaultOption = dayOptions[0];
+
+export const timeOptions = [
```

Comment:
this was just so I had some times available, it should ideally go until 2am because of late night